### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 24

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#350] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325669-d4647c96-8495-4ca2-b675-774e658efd9e.png)

**Comment by i0nut02** [comment]
<img width="304" alt="Schermata 2022-02-08 alle 00 11 27" src="https://user-images.githubusercontent.com/99051485/152887849-c9456c98-b525-4f51-b082-5c6209003a03.png">
